js/core.js

```diff
@@ -1780,6 +1796,30 @@
         this.table = table;
 
         /**
+         *
+         * @type {Object}
+         */
+        this.rights = jsonColumn.rights;
+
+        /**
+         * Mentions whether we should hide the value for this column
+         * @type {Boolean}
+         */
+        this.isHidden = this.rights.select === false;
```

Comment:
It seems like you are not using this variable in the implementation of ermrestjs. It's only being used in test cases.

Reply:
I was using these variables before when I had the select=false handling. But after discussing things with Karl I removed the logic and forgot to remove these variables.

This variable will be useful when I need to remove the column during introspection.